docs/operations_guide/Flink.md

```diff
@@ -29,6 +29,44 @@ In particular, one must not forget that the Flink connector (when checkpoints ar
 * Commits the offsets to Kafka only during checkpoint - so offsets returned by Kafka almost always will not be correct.
 * Ignore offsets in Kafka when it’s started with the checkpointed state - topic offsets are also saved in the checkpointed state.
 
+#### EXACTLY_ONCE delivery mode
+
+The mode is well described here: [this section of Kafka connector documentation](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/connectors/datastream/kafka/#fault-tolerance).
+In order to configure the mode you need to:
```

Comment:
This list is quite long. Maybe we should tweak it a bit by changing the form to a checklist grouped by areas e.g.
In order to achieve End-to-end Exactly once processing/semantics, you need to check multiple places of  configuration:
- Nusssknacker configuration: ...
- Flink cluster configuration: ... 
- Kafka cluster configuration: ...
- Application consuming Nussknacker's messages configuration: ...